### PR TITLE
fix: search and replace URLs with old repo name

### PR DESCRIPTION
**Pull request by cmur2**
## Description

Find usages of the old `camunda/zeebe` repo name in GitHub URLs and replace with `camunda/camunda`.

## Related issues

Related to #18206

### CODE REVIEW

**Review by ChrisKujawa — APPROVED** [review]
LGTM 👍🏼

**Comment by Zelldon** [comment]
🔧 One thing @cmur2 might be worth to split the commit into smaller commits, this would allows us to easily rollback some changes. For example for go client might be worth to do, as we are not sure here about the changes :)

**Comment by cmur2** [comment]
> 🔧 One thing @cmur2 might be worth to split the commit into smaller commits, this would allows us to easily rollback some changes. For example for go client might be worth to do, as we are not sure here about the changes :)

Good point, I split the commit into 4 different ones where 1 is for the Go client.

File: clients/go/go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/camunda/zeebe/clients/go/v8
+module github.com/camunda/camunda/clients/go/v8
```
[the remark this answers is not in the document]
Comment (Zelldon) [reply clients/go/go.mod:1]: Yes this was a conscious business decision, see project channel or this doc https://docs.google.com/document/d/1svn14wH1niB4ihr3OVhglf7qXjjcw5ZzVLzVGgQxEHE/edit?usp=drivesdk :)

**Pull request merged**